test/e2e/cluster.go

```diff
@@ -120,7 +120,7 @@ func createStatefulSet(ctx context.Context, cli kubernetes.Interface) error {
 	installVersion, _ := version.ParseVersion(*oc.ClusterProfile.Version)
 
 	defaultStorageClass := "managed-premium"
-	if installVersion.V[0] == 4 && installVersion.V[1] == 11 {
+	if installVersion.V[0] == 4 && installVersion.V[1] >= 11 {
```

Comment:
Any reason we aren't using the `version` package `Lt` functions? (or create a `Gt` function and use that)

Something like

```golang
	if v.Lt(version.NewVersion(4, 11)) {

Reply:
PR updated